docs/DeveloperGuide.adoc

```diff
@@ -312,6 +312,74 @@ image::ModeCommandActivity.png[]
 ** Cons: Requires proper handling of individual data structures to ensure each list is maintained and updated correctly.
 //end::modeCommand[]
 
+// tag::newsFeedFeature
+=== News Feed Feature
+As people who really love food, we are interested in keeping up-to-date with the latest food trends and newest eateries to try.
+The news feed allows the user to save their favorite food blogs to EatMe and have access to the latest food-related articles right from within the app.
+Should a food blog become irrelevant, the user also has the option to remove it and no longer its articles.
+
+==== Terminology
+Some terms are used in this feature:
+
+- `Feed` - A food blog
+- `Feed Post` - An article from a feed
```

Comment:
Since you are adding terminology, you might also want to include new feed panel, as user wont know the term.